challenge_3/python/returnlove/src/majority_element.py

```diff
@@ -0,0 +1,28 @@
+"""
+To find the majority elements in a list.
+	Counts number of times each element is present in the list
+	prints all the elements which are present more than half of the list
+"""
+
+import math
+
+def find_majority_element(input_list):
+	majority_condition = math.floor(len(input_list)/2)
+	print(majority_condition)	
+	my_dict = {}
+	for element in input_list:
+		if element not in my_dict.keys():
+			my_dict[element] = 1
+		else:
+			my_dict[element] += 1
+
+	for k in my_dict.keys():
+		if my_dict[k] > majority_condition:
+			print(k)
```

Comment:
@returnlove If you did return it, you're right, it would mess up your solution. However, part of the problem is to get around that. For this one, it's not too big a problem. However, for future problems, I think it will important to return, rather than print, the answer. It's not to avoid the use of print, but rather it's to be able to unit test. If you're printing, rather than returning, to test whether or not your function works, you have to visually inspect it. If you're unit testing, you can automate that which is important for larger/more complicated problems.